chrome/content/zotero/xpcom/itemTreeView.js

```diff
@@ -664,6 +656,10 @@ Zotero.ItemTreeView.prototype.notify = Zotero.Promise.coroutine(function* (actio
 			sort = true;
 		}
 		
+		if (collectionTreeRow.isFeed()) {
+			this._ownerDocument.defaultView.ZoteroItemPane.setToggleReadLabel();
+		}
+		
```

Comment:
This actually isn't quite right — as long as the full Zotero for Firefox exists, it's possible to have multiple windows open (and we could conceivably allow that in Standalone at some point). Also possible it would fail if in feed pane with no items selected (again for multiple windows). Should check the currently selected row (if there's only one) against the modified ids to know whether to toggle.

(Also the `invalidateRow()` thing, which it seems like you had before above but removed.)